composites/SnippetPreview/components/ProgressBar.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import colors from "../../../style-guide/colors.json";
+
+const ProgressBarContainer = styled.progress`
+	box-sizing: border-box;
+	width: 100%;
+	height: 8px;
+	display: block;
+	margin-top: 5px;
+	border: none;
+	-moz-appearance: none;
+	-webkit-appearance: none;
+	appearance: none;
```

Comment:
styled-components processes the provided styles already adding prefixes where necessary. If we specify prefixed properties explicitly, the result is duplicated properties:

![screen shot 2017-08-02 at 10 51 37](https://user-images.githubusercontent.com/1682452/28867499-9debacb6-7776-11e7-89c5-9fe94d6024ef.png)

Please remove the prefixed properties.